composer/core/logging/logger.py

```diff
@@ -67,8 +90,18 @@ def __init__(
             self,
             state: State,
             destinations: Sequence[LoggerDestination] = tuple(),
+            run_name: Optional[str] = None,
     ):
         self.destinations = destinations
+        if run_name is None:
+            # prefixing with the time so experiments sorted alphabetically will
+            # have the latest experiment last
+            run_name = str(time.time_ns()) + "-" + coolname.generate_slug(2)
```

Comment:
`time_ns` is definitely more resolution than we need. The point of the `coolname`s is to make these names more readable, but that gets drowned out by an excessively long number up front.

So, two suggestions to make this shorter:
1) Use seconds as resolution. Conflicts between runs should be rare, in which case the coolnames can disambiguate.
2) Encode the time in base 64. As you said, the time is only there for lexicographical sorting, anyways.

Reply:
Ah, you caught my inner 6.172, where code runs so fast one must use nanoseconds haha.

I went with the first suggestion; using `int(time.time())`.